main.go

```diff
@@ -41,29 +45,38 @@ func main() {
 	discord.AddHandler(handleVoiceStateUpdate)
 	// Keep the bot running
 	log.Println("Bot is now running. Press Ctrl+C to exit.")
+	dbUsername := os.Getenv("DB_USER")
+	dbHost := os.Getenv("DB_HOST")
+	dbDB := os.Getenv("DB_DB")
+	dbPassword := os.Getenv("DB_PASS")
+	dbPort := os.Getenv("DB_PORT")
+	dsn := fmt.Sprintf("user=%s password=%s host=%s port=%s dbname=%s sslmode=disable", dbUsername, dbPassword, dbHost, dbPort, dbDB)
+	db.OpenDatabase(dsn)
```

Comment:
Check the error result here and panic if its not nil so we don't startup in an invalid state.